src/chart/helper/EffectSymbol.ts

```diff
@@ -38,7 +36,8 @@ interface RippleEffectCfg {
     zlevel?: number
     symbolType?: string
     color?: ZRColor
-    rippleEffectColor?: ZRColor
+    rippleEffectColor?: ZRColor,
+    effectCount?: number
```

Comment:
I think it will be slightly better if we call it `rippleNumber` here and in the options. From my aspect, the name count is more statistical.

Reply:
Good name! Now it's `number`, just like `period`, `scale`, all of them have no prefix `ripple`. Should we add the prefix for the new option `number`?

Reply:
Seems only `number` is fine.